util/Migrator/DbScripts/2023-02-06_00_ProviderOrganizationProviderDetails.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR ALTER VIEW [dbo].[ProviderOrganizationProviderDetailsView]
```

Comment:
Is there a reason you're not just returning the `Provider` entity? It looks like most of the information just comes from the `Provider` table, or it's information we already have (i.e. the `OrganizationId`).

Could we achieve the same thing by adding a new sproc, e.g. `Provider_ReadByOrganizationId.sql`?

Reply:
I had not considered that approach and it makes way more sense. I've implemented it, thanks for the suggestion.

Reply:
Much nicer 😁